packages/boost.rb

```diff
@@ -3,15 +3,15 @@
 class Boost < Package
   description 'Boost provides free peer-reviewed portable C++ source libraries.'
   homepage 'http://www.boost.org/'
-  version '1.59.0'
-  source_url 'http://sourceforge.net/projects/boost/files/boost/1.59.0/boost_1_59_0.tar.gz'
-  source_sha256 '47f11c8844e579d02691a607fbd32540104a9ac7a2534a8ddaef50daf502baac'
+  version '1.64.0'
+  source_url 'https://downloads.sourceforge.net/project/boost/boost/1.64.0/boost_1_64_0.tar.bz2'
+  source_sha256 '7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332'
 
   def self.build
-    system './bootstrap --prefix=/usr/local'
+    system './bootstrap.sh --prefix=/usr/local'
   end
 
   def self.install
-    system './b2 install'
+    system './b2 -a'
```

Comment:
Sure this is correct? I just tried it with `./b2 install --prefix/usr/local` and it worked perfectly.